runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/ParDoTranslation.java

```diff
@@ -197,7 +199,7 @@ public boolean canTranslate(PTransform<?, ?> pTransform) {
                     ((KvCoder) mainInput.getCoder()).getKeyCoder(),
                     // TODO: Add support for timer payloads to the SDK
                     // We currently assume that all payloads are unspecified.
-                    Timer.Coder.of(VoidCoder.of())));
+                    Timer.Coder.of(StringUtf8Coder.of(), GlobalWindow.Coder.INSTANCE)));
```

Comment:
I chose `StringUtf8Coder` for convenience since there is no place actually parsing this key. I plan to plump through actual coders when migrating to use data channel.

Reply:
sg